lib/instrument/transformer/nskeyed.js

```diff
@@ -108,26 +108,27 @@ class DTTapMessagePlist extends BaseArchiveHandler {
 class NSError extends BaseArchiveHandler {
   decodeArchive (archive) {
     return {
-      '$class': 'NSError',
-      'domain': archive.decode('NSDomain'),
-      'userinfo': archive.decode('NSUserInfo'),
-      'code': archive.decode('NSCode')
+      $class: 'NSError',
+      domain: archive.decode('NSDomain'),
+      userinfo: archive.decode('NSUserInfo'),
+      code: archive.decode('NSCode')
     };
   }
 }
 
 class NSException extends BaseArchiveHandler {
   decodeArchive (archive) {
     return {
-      '$class': 'NSException',
-      'reason': archive.decode('NS.reason'),
-      'userinfo': archive.decode('userinfo'),
-      'name': archive.decode('NS.name')
+      $class: 'NSException',
+      reason: archive.decode('NS.reason'),
+      userinfo: archive.decode('userinfo'),
+      name: archive.decode('NS.name')
     };
   }
 }
 
 class NSURL extends BaseArchiveHandler {
+  static isNSObject = true;
```

Comment:
I'm not quite convinced about having this static field? Would it be possible to override the corresponding method instead and make it in more "polymorphic" manner?

Reply:
I tried to use `_additionClasses` property which is default `[]` in `BaseArchiveHandler`. Make `this._additionClasses = ['NSObject'];` in some of the constructor. But this should be a value just related with class rather than a object. So I changed into this way.
Any better idea?

Reply:
How about something like that:

```
class BaseArchiveHandler {
...
  get isNsObject() {
    return false;
  }
}
....
class NSURL extends BaseArchiveHandler {
  get isNsObject() {
    return true;
  }
}

...

archiveObj.$class = this.uidForArchiver(objName, archiveObj.isNsObject ? 'NSObject' : undefined)
```

Reply:
I changed the code to another way. Please take a look.
Though I still think that a class-only property should not be a non-static class member